src/pages/install/{installConfig.agentName}.js

```diff
@@ -69,15 +70,17 @@ const InstallPage = ({ data, location }) => {
     }
   };
 
-  const handleAgentConfigChange = ({ name, value }) => {
-    tessen.track({
-      eventName: 'agentConfigFileUpdated',
-      category: `${capitalize(name)}AgentConfigFileUpdated`,
-      key: name,
-      value,
-      path: location.pathname,
-      agentName,
-    });
+  const handleAgentConfigChange = ({ name }) => {
```

Comment:
If a user types in half of a name, then comes back to finish the name afterwards, would this fire the full name or the half name? probably nothing we could control...

Reply:
oh! in this case name is the name of the input so "licenseKey" or "applicationName", not dynamic for the end user

Reply:
ahhh totally makes sense